test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -1126,12 +1324,12 @@ class RepresenteeControllerSpec
               mockStoreSession(session.copy(journeyStatus = Some(newJourney)))(Right(()))
             }
 
-            checkIsRedirect(performAction(formData(cgtRef)), routes.RepresenteeController.checkYourAnswers())
+            checkIsRedirect(performAction(formData(cgtRef)), routes.RepresenteeController.confirmPerson())
```

Comment:
this should still point to check your answers - the cya endpoint should then know how to redirect the user to the confirm person page

Reply:
Done.